### ISSUE TITLE
Execution failed for task ':device_apps:verifyReleaseResources' when building APK

### ISSUE THREAD

**Issue by asyrafdayan** [g123k/flutter_plugin_device_apps#17]
```
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /Users/user/Development/Flutter Projects/TestApp/build/device_apps/intermediates/res/merged/release/values/values.xml:236: error: resource android:attr/fontVariationSettings not found.
  /Users/user/Development/Flutter Projects/TestApp/build/device_apps/intermediates/res/merged/release/values/values.xml:237: error: resource android:attr/ttcIndex not found.
  error: failed linking references.   
```
This error message appeared when i was building the APK. 
Anything I can do to not encounter this error?

The version I am currently using is `1.0.8`
My app has also been migrated to AndroidX with `compileSdkVersion 28`

**Comment by mohitjadav** [comment]
@asyrafdayan  same issue here did you find any solution

**Comment by asyrafdayan** [comment]
@mohitjadav Sadly I did not find a solution to this issue and have decided to just remove this package from my project as there have been no comments from the developers of this package

**Comment by g123k** [comment]
For your specific issue @asyrafdayan, it should be fixed in the latest release (1.0.9).
And this error is different from yours @mohitjadav

**Comment by asyrafdayan** [comment]
Thank you for responding. I will be closing down this issue since it appears to have been fixed by the dev.
